app/src/main/java/chat/rocket/android/model/ddp/User.java

```diff
@@ -22,12 +22,13 @@
   private String status;
   private double utcOffset;
   private RealmList<Email> emails;
+  private Settings settings;
```

Comment:
I think it is better to have `settings` with String type (just JSON string).

Realm automatically creates a new record of settings and preference for each updating the user logged in.

![image](https://cloud.githubusercontent.com/assets/11763113/21287511/9217ed24-c4b0-11e6-9b6e-b62339d54a13.png)

For avoiding this, we have to set `@PrimaryKey` into settings and preference. But it is impossible...

So keeping settings as raw string seems to be an only way to prevent increasing records.